src/api/pkg/config/config.go

```diff
@@ -0,0 +1,65 @@
+package config
```

Comment:
This package takes care of loading the `config.yaml` file with the values and overrides depending on the ENVIRONMENT env variable